modules/ai/assets/js/editor/hooks/use-upgrade-message.js

```diff
@@ -0,0 +1,17 @@
+import useIntroduction from './use-introduction';
+
+export const USAGE_PERCENTAGE_THRESHOLD = 80;
+
+const useUpgradeMessage = ( { usagePercentage, hasSubscription } ) => {
+	const { isViewed, markAsViewed } = useIntroduction( 'e-ai-upgrade-message' );
```

Comment:
```suggestion
	const { isBannerViewed, markBannerAsViewed } = useIntroduction( 'e-ai-upgrade-message' );
```

maybe this will help a little bit to understand the conditions below

Reply:
Done.